lib/schemaUtils.js

```diff
@@ -3414,6 +3417,7 @@ module.exports = {
       case 'fallback' : {
         // operationId is usually camelcase or snake case
         expectedReqName = schemaPath.summary || utils.insertSpacesInName(schemaPath.operationId) || reqUrl;
+        typeof expectedReqName === 'string' && (expectedReqName = expectedReqName.substring(0, 255));
         reqNameMismatch = (actualReqName !== expectedReqName);
         break;
       }
```

Comment:
Won't the same issue apply to `url` requestNameSource?

Reply:
@diyaeninan For URL, we are not matching the entire name and just checking if the ending of the request name matches with `reqUrl`. Although here we're comparing ending of trimmed request name instead of actual name so I'll update that handling.